fixtures/martin/mac-viper-airfx.json

```diff
@@ -9,6 +9,10 @@
   },
   "comment": "Viper Air FX:\n- 35000 Lumens \n- 1:5 zoom - Continuous and fast zoom with auto-linked focus\n- 160 mm precision PC front lens for fat beam looks\n- The Martin Quadray Module is available as an accessory for the MAC Viper AirFX. It splits the MAC Viper AirFX beam into four individually controllable rays.",
   "manualURL": "http://www.martin.com/files/files/productdocuments/11_MANUALS/999/UM_MACViperAirFX_EN_B.pdf",
+  "rdm": {
+    "modelId": 44,
+    "softwareVersion": "1.2.0"
```

Comment:
You want to store the version number in DEVICE_INFO, this is just a user friendly string, which may be handy to have, but any matching should be done on the raw number, not the string.

Reply:
I'd like to avoid storing both the nice string and the more "correct" number because you already do so. However, only specifying the number on our side is not very user-friendly or self-explanatory. Since we store the software version only to make sure the modes are not described for an older version, I guess the string works okay for that.

Reply:
Fair enough. I wonder how many people don't update the string when they should. Although there's probably the same risk with the number too, so perhaps it's a bit irrelevant.